website/src/libs/api-v3/amazonPayments.js

```diff
@@ -0,0 +1,93 @@
+import amazonPayments from 'amazon-payments';
+import nconf from 'nconf';
+import common from '../../../../common';
+let t = common.i18n.t;
+const IS_PROD = nconf.get('NODE_ENV') === 'production';
+
+let api = {};
+
+function connect (amazonPayments) { // eslint-disable-line no-shadow
+  return amazonPayments.connect({
+    environment: amazonPayments.Environment[IS_PROD ? 'Production' : 'Sandbox'],
+    sellerId: nconf.get('AMAZON_PAYMENTS:SELLER_ID'),
+    mwsAccessKey: nconf.get('AMAZON_PAYMENTS:MWS_KEY'),
+    mwsSecretKey: nconf.get('AMAZON_PAYMENTS:MWS_SECRET'),
+    clientId: nconf.get('AMAZON_PAYMENTS:CLIENT_ID'),
+  });
+}
+
+api.getTokenInfo = (token) => {
```

Comment:
in this file, using `Promise` directly seems a bit of an overkill to me. Can't we use Q's method for this? This way there's simply more code